sqllogictest/src/runner.rs

```diff
@@ -464,17 +464,19 @@ pub struct Runner<D: AsyncDB> {
     labels: HashSet<String>,
 }
 
-impl<D: AsyncDB> Runner<D> {
-    /// Create a new test runner on the database.
-    pub fn new(db: D) -> Self {
+impl<M: MakeConnection> Runner<M> {
```

Comment:
nit: guess `impl<D: AsyncDB, M: MakeConnection<Conn = D>>` can make it slightly cleaner?